reusable/images_to_numpy.py

```diff
@@ -0,0 +1,133 @@
+import os
+import cv2
+import argparse
+import numpy as np
+from tqdm import tqdm
+from PIL import Image
+import matplotlib.pyplot as plt
+
+"""
+Script to convert a folder of images into numpy format.
+
+Folder structure for images:
+Dataset:
+- Images
+    - image_01
+    - image_02
+    - image_03
+
+Usage: 
+python images_to_numpy.py --root_data_path "./Dataset" --image_path "Images" --saved_file_name "to_numpy" --visualize yes --visualize_for multiple 
+"""
+
+parser = argparse.ArgumentParser(description= 'Convert images to numpy array')
+parser.add_argument('--root_data_path', type= str, required= True, default = "./Dataset", help = 'Define root dataset path.')
+parser.add_argument('--image_path', type= str, required= True, default= "Images", help = 'Set images path.')
+parser.add_argument('--saved_file_name', type= str, required= True, default= "images_to_nparray", help = 'Define saved file name.')
+parser.add_argument('--visualize', type= str, choices= ['no', 'yes'], default= "no", required= True, help = 'Visualize images.')
```

Comment:
required=False, should be optional if users just wants to get numpy

Reply:
Fixed!